Sources/PointFree/Auth.swift

```diff
@@ -221,7 +221,7 @@ private func refreshStripeSubscription(for user: Models.User) -> EitherIO<Error,
 private func gitHubAuthorizationUrl(withRedirect redirect: String?) -> String {
   return gitHubUrl(
     to: .authorize(
-      clientId: Current.envVars.gitHub.clientId,
+      clientId: Current.envVars.gitHub.clientId.rawValue,
```

Comment:
I wanted to put `GitHub.Client.Id` directly in the route, but adding `import GitHub` to `DerivePartialIsos.swift` leads to `GitHub.User`/`Models.User` ambiguity :(

Not sure if there's an easy fix in Sourcery to prefix types with their module names, or maybe we need to stop overlapping and always prefix our types (`GitHubUser`, etc.).

Reply:
I think `GitHubUser` is good!